magicctapipe/scripts/lst1_magic/semi_automatic_scripts/clusters.py

```diff
@@ -35,6 +38,7 @@ def slurm_lines(queue, job_name, array=None, mem=None, out_name=None):
         "#SBATCH -n 1\n\n",
         f"#SBATCH --output={out_name}.out\n" if out_name is not None else "",
         f"#SBATCH --error={out_name}.err\n\n" if out_name is not None else "",
+        f"#SBATCH --nice={nice_parameter}\n" if out_name is not None else "",
```

Comment:
sorry my mistake, it should be  ```if nice_parameter is not None```

Reply:
You're right, I copied the line above and forgot to replace. Thanks